app/controllers/vendors_controller.rb

```diff
@@ -0,0 +1,51 @@
+class VendorsController < ApplicationController
+  before_action :set_vendor, only: %i[show destroy]
+
+  # GET /vendors or /vendors.json
+  def index
+    @vendors = Vendor.all
```

Comment:
Do you think we should address the coverage dings in follow-up PRs?

Reply:
It looks like you already responded to Aaron about this so feel free to disregard.